code/datums/uplink_item.dm

```diff
@@ -1336,6 +1336,13 @@ var/list/uplink_items = list()
 	item = /obj/item/storage/fancy/cigarettes/cigpack_syndicate
 	cost = 2
 
+/datum/uplink_item/badass/rapid
+	name = "Gloves of the North Star"
+	desc = "These gloves let the user punch people very fast. Does not improve weapon attack speed or the meaty fists of a hulk."
```

Comment:
This description would suggest it doesn't work for hulks, but right now there's no code preventing it from working for them?

Reply:
Yeah, I forgot to remove this part of the description. Our hulk code is super snowflakey, so it would still work with this.